src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover } from '@patternfly/react-core';
+import { Icon } from '@patternfly/react-icons';
+
+import { getHostStatusText } from '../../selectors';
+
+export const IconAndText = ({ icon, text }) => (
+  <span className="co-icon-and-text">
+    <Icon type="pf" name={icon} />
+    {text}
+  </span>
+);
+
+IconAndText.propTypes = {
+  icon: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+};
+
+// Generic success status component
+export const GenericSuccess = ({ host }) => <IconAndText icon="pficon-ok" text={getHostStatusText(host)} />;
```

Comment:
Icon component takes the name of the icon, which is the class without the pficon/fa prefix, so you'll have to update the icon prop:
`<IconAndText icon="ok" text={getHostStatusText(host)} />;`

Same in other IconAndText usages

Reply:
Done